frontend/apps/app/components/ChatbotButton/components/ChatMessage/ChatMessage.tsx

```diff
@@ -31,6 +58,40 @@ export const ChatMessage: FC<ChatMessageProps> = ({
       })
     : null
 
+  // For bot messages, we'll render the markdown content with syntax highlighting
```

Comment:
I thought I wanted to output markdown in user message as well. What do you think?